include location of PyQt and Qsci .sip files in SIPCMD macro

author: Kenneth Hoste (HPC-UGent)
author: Pavel Grochal, Pavel Tománek (INUITS)
--- cmake/SIPMacros.cmake.orig	2025-01-10 18:04:46.647020000 +0100
+++ cmake/SIPMacros.cmake	2025-01-13 18:19:48.697455000 +0100
@@ -140,7 +140,7 @@
       ENDIF( ${CONCAT_NUM} LESS ${SIP_CONCAT_PARTS} )
     ENDFOREACH(CONCAT_NUM RANGE 0 ${SIP_CONCAT_PARTS} )
 
-    SET(SIPCMD ${SIP_BINARY_PATH} ${_sip_tags} -w -e ${_sip_x} ${SIP_EXTRA_OPTIONS} -j ${SIP_CONCAT_PARTS} -c ${CMAKE_CURRENT_BINARY_DIR}/${_module_path} -I ${CMAKE_CURRENT_BINARY_DIR}/${_module_path} ${_sip_includes} ${_configured_module_sip})
+    SET(SIPCMD ${SIP_BINARY_PATH} ${_sip_tags} -w -e ${_sip_x} ${SIP_EXTRA_OPTIONS} -j ${SIP_CONCAT_PARTS} -c ${CMAKE_CURRENT_BINARY_DIR}/${_module_path} -I ${CMAKE_CURRENT_BINARY_DIR}/${_module_path} ${_sip_includes} -I$ENV{EBROOTPYQT5}/share/sip -I$ENV{EBROOTQSCINTILLA}/share/sip ${_configured_module_sip})
     ADD_CUSTOM_COMMAND(
       OUTPUT ${_sip_output_files}
       COMMAND ${CMAKE_COMMAND} -E echo ${message}
